doc/api.rst

```diff
@@ -24,6 +24,7 @@ Plots
     plot_khat
     plot_pair
     plot_parallel
+    plot_pointwise_elpd
```

Comment:
Are we going to have elpd plot, or should we make this a bit simpler?

    plot_elpd